src/docs/Updates/Updates/index.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: 'Changelog'
-order: 2
+order: 3
+description: 'The changelog that contain all changes for every Mozaic releases'
```

Comment:
The changelog contains all the changes for every Mozaic release

Reply:
ok